test/DoctrineBatchUtilsTest/BatchProcessing/SimpleBatchIteratorAggregateTest.php

```diff
@@ -274,15 +271,15 @@ public function testIterationFlushesAtGivenBatchSizes(int $resultItemsCount, int
     }
 
     /**
-     * @return int[][]
+     * @return array<int, array<int, int|string>>
      */
     public function iterationFlushesProvider(): array
     {
         return [
-            [10, 5, 3],
-            [2, 1, 3],
-            [15, 5, 4],
-            [10, 2, 6],
+            [10, 5, "beginTransaction\nflush\nclear\nflush\nclear\nflush\nclear\ncommit\n"],
```

Comment:
Messy approach, but works and is clear :D

Reply:
yeah.....